### PR TITLE
Support for Bls12-377 in Prover app

### PR DESCRIPTION
**Pull request by dtebbs**

### CODE REVIEW

File: src/main/java/prover/Prover.java [hunk 1]
```diff
@@ -36,6 +41,7 @@ public static void main(String[] args) throws IOException {
     options.addOption(new Option("t", "test", false, "Run trivial test to verify setup"));
     options.addOption(new Option("p", "primary-size", true, "Size of primary input (1)"));
     options.addOption(new Option("o", "output", true, "Output file (proof.bin)"));
+    options.addOption(new Option("c", "curve", true, "Curve (bn254a*, bls12-377)"));
```
Comment (AntoineRondelet) [inline src/main/java/prover/Prover.java:44]: Why the `*` in `bn254a*`?

Comment (dtebbs) [reply src/main/java/prover/Prover.java]: It signifies the default value.  I thought it was pretty common, but changed in case it's not clear.

Comment (AntoineRondelet) [reply src/main/java/prover/Prover.java]: Ah I see. Thanks, I didn't realize that meant default value - happy both ways, just wanted to make sure it wasn't a typo.

**Review by AntoineRondelet — APPROVED** [review]
LGTM - thanks @dtebbs

**Pull request merged**